telegram/ext/filters.py

```diff
@@ -24,7 +24,19 @@
 
 from abc import ABC, abstractmethod
 from threading import Lock
-from typing import Dict, FrozenSet, List, Match, Optional, Pattern, Set, Union, cast
+from typing import (
+    Dict,
+    FrozenSet,
+    Iterable,
+    List,
+    Match,
+    Optional,
+    Pattern,
+    Set,
+    Tuple,
+    Union,
+    cast,
```

Comment:
IISC this change is only due to the `Iterable` import, which we don't need

Reply:
And `Tuple` which we need. But without `Iterable` it fits in one line.